library/build.gradle

```diff
@@ -69,9 +70,9 @@ uploadArchives {
     repositories.mavenDeployer {
         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-        repository(url: sonatypeRepo) {
```

Comment:
@serggl if this is in your gradle property file, how about I just add a check to see if the value is defined? It'll be something along the lines of

if property file exists or variable exists in System.env (for travis), then execute the call